app/models/maestrano/connector/rails/concerns/organization.rb

```diff
@@ -53,15 +53,29 @@ def initialize(attributes = nil, options = {})
 
     self.synchronized_entities = {}
     Maestrano::Connector::Rails::External.entities_list.each do |entity|
-      synchronized_entities[entity.to_sym] = {can_push_to_connec: !pull_disabled, can_push_to_external: !push_disabled}
+      begin
+        # Transform entity name from entities_list to class name
+        # ex. :item => Entities::Item
+        clazz = "Entities::#{entity.to_s.titleize.tr(' ', '')}".constantize
+      rescue
```

Comment:
rescuing StandardError can be dangerous as it might hide other issues, but in this specific case I am not 100% sure we should rescue NameError only?

Reply:
I'm not really sure either, I took the pattern from `#displayable_synchronized_entities` in this same file. I believe it's the method right below this one. I think there would only be an error thrown when `#constantize` is called. I think it should just set `clazz = nil` in that case since it is only used to set `can_push_to_connec` and `can_push_to_external`